src/components/Modal.js

```diff
@@ -163,7 +163,7 @@ const styles = {
   },
   content: {
     position: 'relative',
-    maxHeight: 'calc(100% - 140px)',
+    maxHeight: '100%',
```

Comment:
For the default I think we should leave the `-140px`, it accounts for the header and footer elements.